docs/content/README.md

```diff
@@ -38,17 +20,19 @@ The Operate First initiative currently manages two clusters within the MOC envir
 
 ### [EMEA][23]
 
-Operate First deploys the following cluster within the EMEA region.
+The Operate First community deploys the following cluster within the EMEA region.
```

Comment:
I was trying to make a pull request of small changes and not only did that 404 on me with the GH API but it ended up letting my commit directly to the original branch. Sorry! These aren't major edits but should be reviewed by you before committing. Next time we'll see if I can do this correctly for suggesting changes with a PR rather than here in the comments.

Reply:
Updated